src/test/java/com/uber/cadence/workflow/interceptors/SignalWorkflowInterceptor.java

```diff
@@ -47,6 +50,20 @@ public SignalWorkflowInterceptor(
     this.next = Objects.requireNonNull(next);
   }
 
+  /**
```

Comment:
comments are not needed for test classes

Reply:
particularly when they provide less information than the type signature